dapp/next.config.js

```diff
@@ -67,13 +66,24 @@ const config = {
   cssLoaderOptions: {
     url: false,
   },
-  images: {
-    loader: "default",
-    domains: ["localhost", "cmsmediaproduction.s3.amazonaws.com", "cmsmediastaging.s3.amazonaws.com"],
-  },
-  i18n: {
-    locales,
-    defaultLocale: 'en',
+  async redirects() {
```

Comment:
sweet this is a much better location for redirects instead of having it defined in the `pages` directory